changelog/1.19.0.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "1.19.0"
+description: "Released on 05/19/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.19.0.
+
+### Features ✨
+
+- web: Resource quotas have been introduced to Organizations. Resource Quotas define
+  maximum resource utilization applied to each member within the Organization they're
+  applied to. Modify them **Manage > Organizations > Edit Organization**.
+- web: Add site and organization workspace as code templates
+- web: An administrative control over Dev URL access levels was added.
+- web: By default, the CVM option for creating custom environments is now checked
+- web: Improved display and support for errors returned by an IdP during sign in
+- infra: Use stdio for envassets
```

Comment:
Need help writing this changelog (unsure how to word) cc @kylecarbs

Reply:
Thus I think this should:

a) move to Breaking Changes
b) Be reworded to somethign along the lines of:

```
infra: Workspace assets previously located in the filepath /opt/coder, such as code-server and coder-cli, have now been moved to /var/tmp/coder
```

Reply:
This is a breaking change, as some of our customers use that code-server binary to install extensions on their workspace.

We've also mentioned this location a few times in our docs, including the "Images" docs: https://github.com/cdr/docs/search?q=%2Fopt%2Fcoder

Reply:
^ @bpmct :confetti_ball: